compiler/rustc_target/src/spec/tests/tests_impl.rs

```diff
@@ -39,7 +39,6 @@ impl Target {
             &self.post_link_args,
         ] {
             for (&flavor, flavor_args) in args {
-                assert!(!flavor_args.is_empty());
```

Comment:
Instead of removing this, could it change to `assert!(!flavor_args.is_empty() || self.arc == "avr");`? I'm not sure exactly why this check is here (looks like @petrochenkov added it) but it seems worth keeping if this target is the only exception.

Reply:
kk, changed